modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -1252,6 +1308,46 @@ private static long maxRevision(long[] revs, long upperBoundRev) {
         return -1;
     }
 
+    /**
+     * Returns index of minimum revision which must be greater or equal to {@code lowerBoundRev}.
```

Comment:
So the **index** should be greater or equal to {@code lowerBoundRev}, is that correct? ;)